app/presenters/publications/provider_recruitment_performance_report_presenter.rb

```diff
@@ -0,0 +1,50 @@
+module Publications
+  class ProviderRecruitmentPerformanceReportPresenter < SimpleDelegator
+    def cycle_name
+      "#{recruitment_cycle_year - 1} to #{recruitment_cycle_year}"
+    end
+
+    def next_cycle_name
+      "#{recruitment_cycle_year} to #{recruitment_cycle_year + 1}"
+    end
+
+    def next_reporting_start_date
+      year = recruitment_cycle_year + 1
+      RecruitmentCycleTimetable.find_by(recruitment_cycle_year: year)
+        .cycle_week_date_range(RecruitmentPerformanceReportTimetable::FIRST_CYCLE_WEEK_REPORT)
+        .first
+        .to_fs(:govuk_date)
+    end
+
+    def report_starting_date
+      recruitment_cycle_timetable
+        .cycle_week_date_range(RecruitmentPerformanceReportTimetable::FIRST_CYCLE_WEEK_REPORT)
+        .first
+        .to_fs(:govuk_date)
+    end
+
+    def report_show_data_from_date
+      recruitment_cycle_timetable.find_opens_at.to_date.to_fs(:govuk_date)
+    end
+
+    def report_show_data_to_date
+      recruitment_cycle_timetable.find_closes_at.to_fs(:govuk_date)
+    end
+
+    def last_cycle_start_date
+      year = recruitment_cycle_year - 1
+      RecruitmentCycleTimetable
+        .find_by(recruitment_cycle_year: year)
```

Comment:
I think this is what I was suggesting earlier - it'd be nice to have 
```suggestion
      recruitment_cycle_timetable.previous_cycle_timetable
```

Reply:
Slightly different from what you suggested in our earlier discussion. if we have a class method `previous_timetable` method that, works like `current_timetable`, but `second_to_last`, that wouldn't work here. Here we are looking for the one that is before the recruitment cycle timetable associated with the report. So if it's a 2024 report, we are looking for 2023, which is the second, second_to_last. But I agree, we could have a `previous_timetable` method that acts on an existing timetable.